hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneFileSystem.java

```diff
@@ -91,6 +93,9 @@ public class BasicOzoneFileSystem extends FileSystem {
   static final Logger LOG =
       LoggerFactory.getLogger(BasicOzoneFileSystem.class);
 
+  private static int listingPageSize =
+      OZONE_CLIENT_FS_LISTING_PAGE_SIZE_DEFAULT;
```

Comment:
Please change this to not be `static`, as findbugs complains about it.

Reply:
Thanks @adoroszlai for reviewing this PR. I'll run all checks locally now before pushing. I didn't aware of this earlier. Thanks for guiding me. Please re-review. Its fixed and passing all checks